src/app/services/sms/__tests__/sms_count.server.model.spec.ts

```diff
@@ -278,7 +284,40 @@ describe('SmsCount', () => {
         'createdAt',
         '__v',
       ])
-      expect(actualSavedObject).toEqual(omit(smsCountParamsWithExtra, 'extra'))
+      expect(actualSavedObject).toEqual(expected)
+    })
+
+    it('should save successfully and set isOnboarded to false when sms is sent using default credentials', async () => {
```

Comment:
should we have test for isOnboarded being true

Reply:
could i clarify this point - i edited the [initial test](https://github.com/opengovsg/FormSG/pull/2276/files#diff-8bc32fb6c3b559254ada4bcadce8e406624db2a61d3f24d5e85cf93b36ae662aR266-R287) so that `isOnboarded` is tested as being part of the success case. were you thinking of something else?

Reply:
I think we should separate the tests into default -> false, custom -> true tests

Reply:
could i clarify this - do you mean another `describe` block nested so that the true/false behaviour is for that block or to have the tests such that the default behaviour is `false` and then we test for `true` (thinking the latter one right)